File: TumCampusApp/src/de/tum/in/tumcampusapp/models/Group.java
package de.tum.in.tumcampusapp.models;

import java.io.Serializable;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

/**
 * Group that describes some organizational entity a person/employee can belong to. Note: This model is based on the
 * TUMOnline web service response format for a corresponding request.
 * 
 * @author Vincenz Doelle
 * @review Daniel G. Mayr
 * @review Thomas Behrens
 */
@Root(name = "gruppe", strict = false)
public class Group implements Serializable {

	private static final long serialVersionUID = -3430225489964912473L;

	@Element(name = "beschreibung", required = false)
	private String description;

	@Element(name = "kennung")
	private String id;

	@Element(name = "org")
	private String org;

	@Element(name = "titel", required = false)
	private String title;

	public String getDescription() {
		return description;
	}

	public String getId() {
		return id;
	}

	public String getOrg() {
		return org;
	}

	public String getTitle() {
		return title;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public void setId(String id) {
		this.id = id;
	}

	public void setOrg(String org) {
		this.org = org;
	}

	public void setTitle(String title) {
		this.title = title;
	}

}

File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/Group.kt
package de.tum.`in`.tumcampusapp.models.tumo

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

import java.io.Serializable

/**
 * Group that describes some organizational entity a person/employee can belong
 * to. Note: This model is based on the TUMOnline web service response format
 * for a corresponding request.
 */

@Root(name = "gruppe", strict = false)
data class Group(@field:Element(name = "beschreibung", required = false)
                 var description: String = "",
                 @field:Element(name = "kennung")
                 var id: String = "",
                 @field:Element(name = "org")
                 var org: String = "",
                 @field:Element(name = "titel", required = false)
                 var title: String = "") :
        Serializable {
    companion object {
        private const val serialVersionUID = -3430225489964912473L
    }
}